nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -139,3 +132,81 @@ def get_weight_config(config: QuantizerConfig, model: onnx.ModelProto) -> Quanti
             )
 
         return config
+
+
+class ONNXQuantizerLayerParameters:
```

Comment:
Is it possible to use `@dataclass` here?

Reply:
I actually did the same in https://github.com/openvinotoolkit/nncf/pull/1410